test-requirements.txt

```diff
@@ -4,3 +4,5 @@ scikit-learn>=0.19.1
 scipy>=0.19.1
 torch>=1.1.0
 pytest
+mdf-connect-client>=0.3.8
+funcx==0.0.6a5
```

Comment:
Why are we pinning the version of FuncX?

Reply:
Not exactly sure myself -- I saw that this was the logic used in foundry with that requirements.txt -- should I unpin it?